app/views/notes/_notes.html.erb

```diff
@@ -65,4 +65,3 @@
     <% end %>
     </div>
 </div>
-<%= will_paginate notes, :renderer => BootstrapPagination::Rails unless @unpaginated %>
```

Comment:
Yeah makes sense. But I am having trouble using the partials with pagination, because I have four different items on the same page..i.e notes, wikis, questions and profiles. And they are all trying to paginate the same page. But will try to find a work-around on that and if you have any pointers please let me know.